src/carousel.js

```diff
@@ -59,9 +68,9 @@ class Carousel extends Component {
   }
 
   scrollToIndex(index) {
```

Comment:
it should not allow to scroll to `index=0` or `index=data.length-1` when `alwaysCentralizeSelected` is true.